modules/artemis/example_test.go

```diff
@@ -0,0 +1,34 @@
+package artemis_test
+
+import (
+	"context"
+
+	"github.com/go-stomp/stomp/v3"
+	"github.com/testcontainers/testcontainers-go/modules/artemis"
+)
+
+func ExampleRunContainer() {
+	ctx := context.Background()
+
+	user := "username"
+	pass := "password"
+
+	// runContainer {
+	container, err := artemis.RunContainer(ctx, artemis.WithCredentials(user, pass))
+	if err != nil {
+		panic(err)
+	}
+	// }
+
+	host, err := container.BrokerEndpoint(ctx)
+	if err != nil {
+		panic(err)
+	}
+
+	conn, err := stomp.Dial("tcp", host, stomp.ConnOpt.Login(user, pass))
+	if err != nil {
+		panic(err)
+	}
+
+	_ = conn
```

Comment:
Do we want to terminate the container here in the example to make the code snippet self-explanatory?

Reply:
Done